### PR TITLE
Gtests for Utilities.cpp

### PR DESCRIPTION
**Pull request by SgStapleton**
Tests for Utilities.cpp as well as refactoring of method to return by parameter.

Also, removed unneeded focalBias and halfSwath parameters as per @jessemapel guidance.

### CODE REVIEW

**Comment by SgStapleton** [comment]
Should I get tests going for all of the functions added over the weekend?

**Comment by SgStapleton** [comment]
This finishes-up https://github.com/USGS-Astrogeology/usgscsm/issues/165

File: tests/LineScanCameraTests.cpp [hunk 1]
```diff
@@ -126,3 +126,11 @@ TEST_F(ConstAngularVelocityLineScanSensorModel, OffBody4) {
...
    // EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
 }
+
+// TEST_F(ConstVelocityLineScanSensorModel, calculateAttitudeCorrection) {
+//    // csm::ImageCoord imagePt(8.5, 8.0);
+//    // csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
+//    // EXPECT_DOUBLE_EQ(groundPt.x, 10.0);
+//    // EXPECT_DOUBLE_EQ(groundPt.y, 0.0);
+//    // EXPECT_DOUBLE_EQ(groundPt.z, 0.0);
+// }
```
Comment (jessemapel) [inline tests/LineScanCameraTests.cpp]: Is this leftover from something

File: tests/UtilitiesTests.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+   calculateRotationMatrixFromEuler(euler, rotationMatrix);
+   EXPECT_NEAR(rotationMatrix[0], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[1], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[2], 1, 1e-8);
+   EXPECT_NEAR(rotationMatrix[3], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[4], 1, 1e-8);
+   EXPECT_NEAR(rotationMatrix[5], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[6], -1, 1e-8);
+   EXPECT_NEAR(rotationMatrix[7], 0, 1e-8);
+   EXPECT_NEAR(rotationMatrix[8], 0, 1e-8);
```
Comment (jessemapel) [inline tests/UtilitiesTests.cpp]: These should be EXPECT_DOUBLE_EQ and not EXPECT_NEAR

**Comment by kberryUSGS** [comment]
:+1: This looks good to me aside from the `EXPECT_DOUBLE_EQ` change Jesse already pointed out!

File: tests/UtilitiesTests.cpp (hunk 2 above)
Comment (SgStapleton) [reply tests/UtilitiesTests.cpp]: EXPECT_NEARs are needed for precision issue here. Added comments to explain as much.

**Comment by SgStapleton** [comment]
Confirmed this is building and passing all tests on prog27. I am not sure what is up with the travis CI...

**Pull request merged**